modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/SCIM2RoleTestCase.java

```diff
@@ -0,0 +1,591 @@
+package org.wso2.identity.integration.test.scim2;
```

Comment:
Add the license header.

Reply:
The license header was not added in the other scim tests as well. That's why I didn't add the license header.

Reply:
It might be a mistake then. We should add the relevant license header to all the source code files.

Reply:
Fixed with 46bc960a257570eaea409e05724ad8f0ffb4d4eb